bin/dev

```diff
@@ -1,9 +1,8 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
-if ! foreman version &> /dev/null
-then
-    echo "Installing foreman..."
-    gem install foreman
+if ! gem list foreman -i --silent; then
+  echo "Installing foreman..."
+  gem install foreman
 fi
```

Comment:
Prior formulation is the outdated cssbundling way of doing things (which installs foreman on every single invocation of `bin/dev`). Updated to latest way because less annoying.